MODERATION_POLICY.md

```diff
@@ -0,0 +1,297 @@
+
+### Collaborator Posts
+
+* Prior to Moderating any Post authored by a Collaborator, the author must be
+  given a reasonable opportunity to modify or remove the Post on their own.
+* If the author of the Post disagrees that Moderation is required, the matter
+  can be escalated to the TSC for resolution. In such cases, no Moderation
+  action may be taken until a decision by the TSC is made.
+* When Moderating any Post authored by another Collaborator, the moderating
+  Collaborator must:
+  * Explain the justification for Moderating the post,
+  * Identify all changes made to the Post, and
+  * Identify the steps previously taken to resolve the issue.
+  * If the Moderation action included Blocking, an indication of whether the Block
+    is temporary or indefinite is required, along with an issue posted to the
+    moderation repository justifying the action.
+* Explanations of Moderation actions on Collaborator Posts must be provided in:
+  * A new post within the original thread, or
+  * A new issue within the private webpack/moderation repository.
+* Any Collaborator habitually violating the Code of Conduct or this Moderation
+  policy may be Blocked temporarily or, in extreme cases, Removed and Blocked
+  indefinitely.
+
+### Non-Collaborator Posts
+
+* Posts authored by non-Collaborators are always subject to immediate Moderation
+by any Collaborator if the content is intentionally disruptive or in violation
+of the [Code of Conduct][].
+* When moderating non-Collaborator posts, the moderating Collaborator must:
+  * Explain the justification for Moderating the post, and
+  * Identify all changes made to the Post.
+  * If the Moderation action included Blocking, an indication of whether the Block
+    is temporary or indefinite is required, along with a note justifying the
+    action.
+* If an explanation of a Moderation action for a non-Collaborator Post is
+  provided, it must be provided in:
+  * The original Post being modified (as replacement or appended content),
+  * A new post within the original thread, or
+  * A new issue within the private webpack/moderation repository.
+* Moderation of Posts authored by non-Collaborators may result in those
+  non-Collaborators being Blocked temporarily or indefinitely from further
+  participation in the webpack GitHub organization.
+* If it is clear that there is no intention to collaborate in good faith,
+  it is possible to hide comments of non-Collaborators. In that case there is
+  an exception to the reporting requirement described above.
+* Accounts that are reasonably believed to be bots (other than bots authorized
+  by the TSC) are subject to immediate Blocking.
+* Issues, pull requests, discussions, and comments that are spam (job posting,
+  service advertising, etc.) are subject to immediate moderation.
+* Issues, pull requests, discussions, and comments that are believed to be
+  LLM-generated (e.g a PR coming from a new contributor changing a single file
```

Comment:
@alexander-akait a note for you about AI content

Reply:
Thanks